clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/CopyLastAsgAtomicOperation.groovy

```diff
@@ -131,6 +131,7 @@ class CopyLastAsgAtomicOperation implements AtomicOperation<DeploymentResult> {
       newDescription.ebsOptimized = description.ebsOptimized != null ? description.ebsOptimized : ancestorLaunchConfiguration.ebsOptimized
       newDescription.classicLinkVpcId = description.classicLinkVpcId != null ? description.classicLinkVpcId : ancestorLaunchConfiguration.classicLinkVPCId
       newDescription.classicLinkVPCSecurityGroups = description.classicLinkVPCSecurityGroups != null ? description.classicLinkVPCSecurityGroups : ancestorLaunchConfiguration.classicLinkVPCSecurityGroups
+      newDescription.tags = description.tags ?: ancestorAsg.tags.collectEntries { [(it.getKey()): it.getValue()] }
```

Comment:
I'm not the right guy to review this PR, but I think you want `description.tags != null` here. That is, if you want an explicitly empty map to cause the newly-provisioned server group to have no tags, as opposed to inheriting the tags of the server group being cloned.